ford/sourceform.py

```diff
@@ -77,8 +78,8 @@
               'contiguous','continue','cos','cosh','count','cpu_time','critical',
               'cshift','cycle','data','ctime','dabs','date_and_time','dble',
               'dcmplx','deallocate','deferred','digits','dim','dimension','do',
-              'while','dlog','dlog10','dmax1','dmin1',
```

Comment:
why did you get rid of while here?

```suggestion
              'while','dlog','dlog10','dmax1','dmin1',
```

Reply:
Looks like `'while'` appears later on and this is removing a duplicate entry